Exercises/Java Scrip/Ex_4/index.js

```diff
@@ -0,0 +1,5 @@
+alert("Welcome!");
+let firstNumber = prompt("Enter First Number");
+let secondNumber = prompt("Enter Second Number");
+let result = confirm(firstNumber >secondNumber);
```

Comment:
To do an accurate number comparison, you should convert the variables to numbers. Otherwise, the comparison will be based on the character codes